Makefile

```diff
@@ -57,6 +57,7 @@ distribution-tarball:
 		--exclude=.github \
 		--exclude=.gitignore \
 		--exclude=.copr \
+		--exclude=development \
```

Comment:
@r0x0d this would easily fix the problem, does the development folder has to be in the distribution tarball? (I have no idea what is general convention)

Also we could just waive the problems in the scan :thinking:

Reply:
The development folder does not need to be in the final tarball, as it is something only related to what we use during development, so, it doesn't need to be in the tarball at all.

Reply:
alright, then that should resolve all problems with development folder because all listed problems seem to work with unpacked tarball e.g. `rhc-worker-script-0.4/development/.env:3:8:`